### DISCUSSION TITLE
tvOS Support

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#4268] [Platforms]
So I wanted to ask about this.  I'm not sure if this is exactly the right place,  but I was looking and I saw that there is a pr for tvOS support,  but I'm new to reading all of this,  and I'm unsure of if the plan is to eventually pull those into the official 3.x main branches.  Are they likely to be added?  And if so is there any testing etc that I could help with to make it happen faster?

**Comment by theraot** [comment]
I don't know about tvOS in particular, don't ask me about the plans for this…

However, if there are pull requests, you can test them: If they are recent you can get the artifacts from GitHub (Go to Checks, select the platform on the left, scroll down to Artifacts on the right, and download them). If the artifacts expired, then clone the repository being pulled and go over the build process on your machine (you can find instructions on the official documentation).

Then you can leave a review (Go over Files Changes and click Review Changes). Approve if you tested it and also went over the code and didn't find any problems, please be mindful with your approvals. If you just tested it but didn't go over the code, or if you are not sure if the code is OK, you can comment instead. If you are sure you found an issue in the code you use the request changes option.

*And I know that reactions (the emoji icons) on the pull request don't mean much, but it does not harm to leave a reaction. It may help to signal interest in the feature.*

However, be aware that an approval does not mean it will be merged right away. And being a new feature, the pull requests for the master branch would have to go first.

**Reply by Calinou** [reply]
See [Testing pull requests](https://docs.godotengine.org/en/latest/community/contributing/testing_pull_requests.html) in the documentation for more information.
